### DISCUSSION TITLE
bw2024读取不到选票信息

### DISCUSSION THREAD

**Discussion by AiAe-156** [mikumifa/biliTickerBuy#157] [提问专区] [answered]
如题，bw2024读不到选票信息，其他随便选一个就能读到
![image](https://github.com/mikumifa/biliTickerBuy/assets/82350631/bfbc3d02-8001-482b-8257-1f599794894d)

**Comment by mikumifa** [comment]
你猜猜为什么读不了

**Comment by duyueovo** [comment, marked as answer]
没开票呢 等官方公布之后才能读到
